src/libs/Navigation/AppNavigator/getRootNavigatorScreenOptions.ts

```diff
@@ -15,6 +15,8 @@ const commonScreenOptions: StackNavigationOptions = {
     animationTypeForReplace: 'push',
 };
 
+const SLIDE_LEFT_OUTPUT_RANGE_MULTIPLIER = -1;
```

Comment:
should we move this to the const file or can we keep this here?

Reply:
I think this variable won't be used elsewhere in the application, it is specific to function modelCardStyleInterpolator which is only called in this file